src/renderer/Presentational/ContentMultipleClients/ContentMultipleClients.tsx

```diff
@@ -175,37 +213,32 @@ const ContentMultipleClients = (props: {
       <HeaderMetrics {...(nodeOverview as NodeOverviewProps)} />
       <HorizontalLine type="content" />
       {renderPrompt()}
```

Comment:
I see the following, so we might need to add a promptContainer div around the prompts, so that there's a marginBottom: 34.

<img width="761" alt="Screen Shot 2023-09-15 at 5 34 21 PM" src="https://github.com/NiceNode/nice-node/assets/99524051/094e62cd-dd3c-43a5-a580-cb93f8e5b712">

sample code:

```
const renderPrompt = () => {
    const synchronized =
      clClient?.status.synchronized && elClient?.status.synchronized;
    let promptComponent = null;
    if (
      synchronized &&
      !clClient?.status.updating &&
      !elClient?.status.updating &&
      !walletDismissed
    ) {
      promptComponent = (
        <WalletPrompt
          onSetupClick={onSetupClick}
          onDismissClick={onDismissClick}
        />
      );
    }
    if (
      !clClient?.status.initialized &&
      !elClient?.status.initialized &&
      !synchronized &&
      !initialSyncMessageDismissed
    ) {
      const title = 'Initial sync process started';
      const description =
        'When adding a node it first needs to catch up on the history of the network. This process downloads all the necessary data and might take a couple of days. After synchronization is complete your node will be online and part of the network.';
      promptComponent = (
        <Message
          type="info"
          title={title}
          description={description}
          onClick={() => {
            localStorage.setItem('initialSyncMessageDismissed', 'true');
            setinitialSyncMessageDismissed(true);
          }}
        />
      );
    }
    return <div className={promptContainer}>{promptComponent}</div>;
  };
```

Reply:
<img width="765" alt="Captura de pantalla 2023-09-18 a la(s) 3 58 37 p m" src="https://github.com/NiceNode/nice-node/assets/3721291/fdd8c4c1-c124-4b5b-9529-b007c165d676">